packages/ui/components/analytics/AnalyticsCard.module.css

```diff
@@ -0,0 +1,163 @@
+.card {
+  border-radius: 16px;
+  background: #f8f8f9;
+  display: flex;
+  padding: 96px 0px;
+  justify-content: center;
+  align-items: center;
+  gap: 40px;
+  flex: 1 0 0;
+}
+.cardGroup {
+  display: flex;
+  align-items: flex-start;
+  gap: 16px;
+  align-self: stretch;
+}
+.cardContent {
+  display: flex;

```

Comment:
Same here, no need to declare similar properties which are already present for parent element.

Reply:
@DeltaDynamo review if this comment is resolved or not

Reply:
@amoghar29  Please use `grouping` in css files to reduce LOC, if possible.
Use this as reference: https://www.geeksforgeeks.org/explain-nesting-and-grouping-in-css/

Reply:
okay.

Reply:
@DeltaDynamo